openassessment/xblock/static/js/spec/studio/oa_edit_assessments.js

```diff
@@ -250,4 +250,25 @@ describe("OpenAssessment edit assessment views", function() {
         it("Loads a description", function() { testLoadXMLExamples(view); });
         it("shows an alert when disabled", function() { testAlertOnDisable(view); });
     });
+
+    describe("OpenAssessment.EditStaffAssessmentView", function() {
+        var view = null;
+
+        beforeEach(function() {
+            var element = $("#oa_stafff_assessment_editor").get(0);
```

Comment:
Hmm, yeah that does seem odd. I would have thought we'd get a failure with this typo.

Regardless, I'll get this fixed.